api/v1alpha1/common_types.go

```diff
@@ -116,6 +120,7 @@ type MetaObject interface {
 
 // +kubebuilder:object:generate=false
 
+// TODO: refactor/cleanup ChaosInstance only used in common controller, and only field Name is used, I think it could be replace with other kind kie GenericChaos in this usage.
```

Comment:
what is the meaning of `kie`? or a typo?

Reply:
yes it's a typo, it should be "like". :D